config/config.go

```diff
@@ -88,6 +89,8 @@ func Parse(c CfgMap) error {
 				}
 
 				reflect.ValueOf(v.Value).Elem().SetBool(val)
+			case reflect.Slice:
+				*(v.Value.(*[]string)) = strings.Split(envValue, ",")
```

Comment:
This doesn't look right, are we able to use `reflect.ValueOf` here? 🤔

Reply:
hey @joshuasing could you check it now? I was able to use `reflect.AppendSlice` for this, wondering what you think?

Reply:
@ClaytonNorthey92 This appears to crash:
```
panic: reflect: call of unknown method on ptr Value

goroutine 1 [running]:
reflect.flag.mustBe(...)
        reflect/value.go:233
reflect.AppendSlice({0x1053e0be0?, 0x1057b4c38?, 0x1053a5260?}, {0x1053e2220?, 0x14000039d50?, 0x104e3f360?})
        reflect/value.go:2954 +0x20c
github.com/hemilabs/heminetwork/config.Parse(0x14000039f38?)
        github.com/hemilabs/heminetwork/config/config.go:94 +0x698
main._main()
        github.com/hemilabs/heminetwork/cmd/tbcd/tbcd.go:127 +0x28
main.main()
        github.com/hemilabs/heminetwork/cmd/tbcd/tbcd.go:194 +0x128
```

Reply:
strings.Split works fine here too.